ingest/monitoring/metrics_service.py

```diff
@@ -0,0 +1,33 @@
+# Service to record events to Mixpanel via Bard
+#
+# Mixpanel is a business analytics service for tracking user interactions.
+# Bard is a DSP service that mediates writes to Mixpanel.
+#
+
+import json
+import requests
+
+
+class MetricsService:
+    BARD_HOST_URL = "https://terra-bard-dev.appspot.com/api/event"
+    user_id = "2f30ec50-a04d-4d43-8fd1-b136a2045079"
```

Comment:
Same thing - why is the `user_id` hard-coded?

Reply:
From my understanding, it doesn't matter. @devonbush can provide more context.